apps/perf-test/tsconfig.json

```diff
@@ -8,14 +8,11 @@
     "declaration": true,
     "sourceMap": true,
     "experimentalDecorators": true,
-    "forceConsistentCasingInFileNames": true,
-    "moduleResolution": "node",
     "preserveConstEnums": true,
     "strictNullChecks": true,
     "noImplicitAny": true,
     "lib": ["es6", "dom"],
-    "types": ["webpack-env"],
-    "skipLibCheck": true
+    "types": ["webpack-env", "node"]
```

Comment:
node is needed because some v8 packages are relying on that global type existence (they are using `NodeJS.*` namespace which is exposed on global if `node` types is present)